Mafeus/ContaCorrente.java

```diff
@@ -0,0 +1,35 @@
+public class ContaCorrente extends Conta{
+
+    public ContaCorrente(int Num, Cliente Dono, double saldo, double limite){
+
+        super(Num, Dono, saldo, limite);
+        setLimite(limite, saldo);
+
+    }
+
+
+
+    void setLimite(double limite, double saldo) {
+
+        if (saldo < -100){
+
+            saldo = -100;
+
+            saldo = super.getSaldo();
+            limite = super.getLimite();
+
+        }
+
+    }
+
+    public double calculaTaxas() {
+
+        if (getDono().getClass() == PessoaFisica.class){
+            return 10;
```

Comment:
Matheus, essa implementação funciona, mas não é a ideal porque não segue as regras do polimorfismo. Polimorfismo permite tratar objetos de diferentes classes de maneira uniforme, sem precisar saber o tipo específico do objeto. Portanto, é necessário acessar diretamente o método calcularTaxas do dono.
Assim: return this.getDono().calcularTaxas().